pkgs/by-name/di/diversion-cli/package.nix

```
@@ -0,0 +1,49 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+  nix-update-script,
+}:
+let
+  srcs = {
+    "x86_64-linux" = {
+      url = "https://archive.org/download/diversion-cli/dv_linux_x86_64_0-7-195";
+      sha256 = "15fDNp4isdgmMkL6g+LVC90S6FItoT29X3cCIqtbxLc=";
+    };
+
+  };
+in
+stdenv.mkDerivation {
+  pname = "diversion-cli";
+  version = "0.7.195";
+  src =
+    assert lib.assertMsg (builtins.hasAttr stdenv.hostPlatform.system srcs)
+      "Diversion CLI is not available for ${stdenv.hostPlatform.system}";
+    fetchurl {
+      url = srcs.${stdenv.hostPlatform.system}.url;
+      sha256 = srcs.${stdenv.hostPlatform.system}.sha256;
+    };
+
+  phases = [ "installPhase" ];
+  installPhase = ''
+    mkdir -p "$out/bin"
+    install -D $src $out/bin/dv
+
+    export PATH="$out/bin:$PATH"
+  '';
```

```suggestion
  installPhase = ''
    runHook preInstall
    mkdir -p "$out/bin"
    install -D $src $out/bin/dv
    runHook postInstall
  '';
```
No need to set $PATH (it only affects the build environment)

---

Fixed in [6fa6eba](https://github.com/NixOS/nixpkgs/pull/372202/commits/6fa6ebacab45966e728a590276cad0f25dfaf41b)